src/garage/torch/_functions.py

```diff
@@ -273,3 +291,36 @@ def product_of_gaussians(mus, sigmas_squared):
     sigma_squared = 1. / torch.sum(torch.reciprocal(sigmas_squared), dim=0)
     mu = sigma_squared * torch.sum(mus / sigmas_squared, dim=0)
     return mu, sigma_squared
+
+
+class TransposeImage(gym.ObservationWrapper):
+    """Transpose observation space for image observation in PyTorch."""
+
+    def __init__(self, env=None):
+        """Transpose image observation space.
+
+        Reshape the input observation shape from (H, W, C) into (C, H, W)
+        in pytorch format.
+
+        Args:
+            env (garage.envs): environment
+        """
+        super(TransposeImage, self).__init__(env)
```

Comment:
You don't actually need to pass any parameters to `super()` in our versions of python.